app/controllers/pages_controller.rb

```diff
@@ -58,7 +58,8 @@ def patrons
              .select("CEIL(100 * LN(COUNT(referees.id) + 1)) AS aura")
              .joins("LEFT JOIN users referees ON users.id = referees.referrer_id")
              .group("users.id")
-             .order(aura: :desc)
+             .having("CEIL(100 * LN(COUNT(referees.id) + 1)) > 0")
```

Comment:
can't you use the `aura` alias set 3 lines above instead of recomputing the aura ?

Reply:
Etonnant 🤔